src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Microsoft.Extensions.AI.Evaluation.Reporting.csproj

```diff
@@ -1,5 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <!-- 
+    NOTE: To build and test the TypeScript code that this project depends on, set the 'EnableEvaluationReportBuild'
+    environment variable to 'true'. Note that the TypeScript build depends on Node.js tools. So you will need to
+    install the latest version of Node.js from https://nodejs.org/ as a prerequisite before setting the
+    'EnableEvaluationReportBuild' environment variable.
+  -->
```

Comment:
Should this be added to the project's readme?

Reply:
Yes, I'm working on adding a section under `docs\building.md`. Not sure if we should add it to `README.md` for the current project - doesn't that get bundled up as part of the NuGet package?

Reply:
I have merged the current PR - I'll create a follow up PR to update the docs.